app/charts/area/areas-state.tsx

```diff
@@ -352,6 +353,9 @@ const useAreasState = (
   xScaleTimeRange.range([0, chartWidth]);
   yScale.range([chartHeight, 0]);
 
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down("sm"));
```

Comment:
Nit: As this logic is used across a few different places in the application already, it would be super-nice to extract to a `useIsMobile` hook :)

Reply:
ah cool I thought so we had something I was searching for useMobile hahaha

Reply:
Sure  👍

Reply:
@bprusinowski could you tell me an ideal place to put it?

Reply:
@noahonyejese '@/utils/use-is-mobile.ts'?